packages/discovery-provider/plugins/verified-uploads/src/handlers/users.js

```diff
@@ -36,41 +36,44 @@ export default async ({ user_id, blocknumber }) => {
     const is_verified = current.is_verified
     const handle = current.handle
 
-    const data = await retry(
-      async (_) => {
-        const { data } = await axios
-          .get(social_handle_url(handle))
-          .catch(console.error)
+    let source: str
```

Comment:
`let source: string`

Reply:
should actually not have a type, this is a js file

Reply:
made fix locally and pushed

Reply:
WOOPS thx